packages/@o3r/rules-engine/src/engine/ruleset-executor.ts

```diff
@@ -195,112 +326,4 @@ export class RulesetExecutor {
     throw new Error(`Unknown condition block met : ${JSON.stringify(nestedCondition)}`);
   }
 
-  /**
-   * Plug ruleset to fact streams and trigger a first evaluation
-   *
-   * @param ruleset
-   */
-  public prepareRuleset() {
```

Comment:
we cannot remove public method like this

Reply:
it wasn't supposed to be public in the first place :) only used within this file

Reply:
But it is now 
we can deprecate it in v9 and set it private in v10

Reply:
Even if it is now, it was not supposed to be. 
The method is only called in the constructor of this class and this class is not exported
<img width="977" alt="image" src="https://github.com/AmadeusITGroup/otter/assets/86055112/fe3cb258-f0a9-467c-b863-9d8151b5ea98">

I moved it back down to have a correct diff on the PR and added a TODO to move it back to top as public in a separate PR.